BillManagementSystem/accounting/schema/schema.py

```diff
@@ -0,0 +1,69 @@
+import strawberry
+from typing import List
+from django.contrib.auth.models import User as AdminUser
+from .inputs import CategoryInput, CategoryPartialInput, RegisterInput, BillInput, BillPartialInput, IndividualSpendingInput, IndividualSpendingPartialInput
+from strawberry_django import mutations
+from .. import models
+from django.contrib.auth.tokens import default_token_generator
+
+from .types import IndividualSpendingScalar, BillScalar, CategoryScalar, UserScalar, AuthResponse
+from .filters import UserFilter, BillFilter, IndividualSpendingFilter, CategoryFilter
+
+from gqlauth.user import arg_mutations as mutationsAuth
+
+@strawberry.type
+class Query:
+    categories: List[CategoryScalar] = strawberry.django.field()
+    category: List[CategoryScalar] = strawberry.django.field(filters=CategoryFilter)
+    
+    users: List[UserScalar] = strawberry.django.field()
+    user: List[UserScalar] = strawberry.django.field(filters=UserFilter)
+
+    bills: List[BillScalar] = strawberry.django.field()
+    bill: List[BillScalar] = strawberry.django.field(filters=BillFilter)
+    
+    individualSpendings: List[IndividualSpendingScalar] = strawberry.django.field()
+    individualSpending: List[IndividualSpendingScalar] = strawberry.django.field(filters=IndividualSpendingFilter)
+    
+    
+    
+@strawberry.type
+class Mutation:
+    
+    # create
+    createCategory: CategoryScalar = mutations.create(CategoryInput)
+    createBill: BillScalar = mutations.create(BillInput)
+    createIndividualSpending: IndividualSpendingScalar = mutations.create(IndividualSpendingInput)
+    
+    # update
+    updateCategory: List[CategoryScalar] = mutations.update(CategoryPartialInput, filters=CategoryFilter)
+    updateBill: List[BillScalar] = mutations.update(BillPartialInput, filters=BillFilter)
+    updateIndividualSpending: List[IndividualSpendingScalar] = mutations.update(IndividualSpendingPartialInput, filters=IndividualSpendingFilter)
+    
+    # delete
+    
+    # TODO:this is dangerous, they can feed in deletion all without filter
```

Comment:
Probably you can add a custom filter / input for deletion that only accepts one and only one id?
Or you can just comment out deletions for now and proceed with the rest ahaha

Reply:
I think I will keep this deletion first. But create this as issue.